src/chordChart.ts

```diff
@@ -1201,10 +1198,12 @@ export class ChordChart implements IVisual {
         d !== null && d.data !== null && d.data.label !== null,
       style: {
         fill: (d: ChordArcDescriptor) => d.data.labelColor,
-        "text-anchor": (d: ChordArcDescriptor) =>
-          midAngle(d) < Math.PI ? "start" : "end",
-        "font-size": () =>
-          PixelConverter.fromPoint(this.settings.labels.fontSize.value),
+        "text-anchor": (d: ChordArcDescriptor) => midAngle(d) < Math.PI ? "start" : "end",
+        "font-size": () => PixelConverter.fromPoint(this.settings.labels.font.fontSize.value),
+        "font-family": () => this.settings.labels.font.fontFamily.value.valueOf() || dataLabelUtils.StandardFontFamily,
```

Comment:
why we need to call a valueOf() here?

Reply:
I confused with `SimpleSlice<T>.valueOf()`. It's not needed here and I'll remove it.